packages/dapp/src/utils/arweave/tasks/connect.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
all files inside of `utils/arweave/tasks/` are mostly so I'm able to test 😅, I'm still trying to understand how arweave works

Reply:
i committed these just because i thought these might be helpful (similar to hardhat tasks), but if we don't want them, I can remove them